src/services/storage/Storage.ts

```diff
@@ -72,7 +72,16 @@ export type SavedKeymapData = {
   keycodes: { [pos: string]: number }[];
   created_at?: Date;
   updated_at?: Date;
-};
+}
+
+export interface SavedKeymapData extends AbstractKeymapData {
+  status: SavedKeymapSatus;
+}
+
+export interface AppliedKeymapData extends AbstractKeymapData {
+  applied_uid: string;
+  saved_keymap_data_id: string;
```

Comment:
I thinks, saved_keymap_id would be good because 'Data' is the suffix words in order to explicit the Firestore entity in the application no in the DB.

Reply:
@adamrocker I change the property name at 018d6cf. Thanks!